src/infinite-scroll/index.ts

```diff
@@ -222,9 +223,13 @@ class InfiniteScroll {
     this.set({ isFetchingForward, isFetchingBackward });
 
   scroll = () => {
-    const { scroller, wrapper } = this.state;
+    const { scroller, wrapper, windowScroll } = this.state;
     const wrapperHeight = wrapper.getBoundingClientRect().height;
-    const scrollY = this.state.windowScroll ? Core.utils.WINDOW().pageYOffset : scroller.root.scrollTop;
+    const windowWrapper = this.state.windowScroll && Core.utils.WINDOW();
```

Comment:
You already included `windowScroll` in an earlier line, so you may as well use it.

```suggestion
    const windowWrapper = windowScroll && Core.utils.WINDOW();
```